CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to Pulumi Examples
+
+Pulumi welcomes contributions from the community and is excited to continue building out its collection of examples and tutorials that showcase the power of modern infrastructure as code. This guide is intended to
+help your contribution get published quickly.
+
+## Pulumi terminology
+
+For a list of Pulumi terms used throughout our documentation site, see the [Glossary page](https://www.pulumi.com/docs/reference/glossary/).
+
+## Example folder structure
+
+The folder structure for each example includes a two-part prefix, `<cloud>-<language>` (to indicate which `<cloud>` and `<language>` it pertains to), and a brief descriptive name with hyphens in between the words.
+
+The cloud prefix can be one of:
+
+- `aws` for [Amazon Web Services](https://github.com/pulumi/pulumi-aws)
+- `azure` for [Microsoft Azure](https://github.com/pulumi/pulumi-azure)
+- `gcp` for [Google Cloud Platform](https://github.com/pulumi/pulumi-gcp)
+- `kubernetes` for [Kubernetes](https://github.com/pulumi/pulumi-kubernetes),
+- `digitalocean` for [DigitalOcean](https://github.com/pulumi/pulumi-digitalocean/)
+- `f5bigip` for [F5's BIG-IP](https://github.com/pulumi/pulumi-f5bigip/)
+- `cloud` for [Pulumi's cross-cloud programming framework](https://github.com/pulumi/pulumi-cloud), which is currently in preview
+- Any [cloud provider](https://www.pulumi.com/docs/reference/pkg/#cloud-providers) with a dedicated Pulumi package
+
+The language prefix can be one of:
+- `ts` for TypeScript
+- `js` for JavaScript
+- `py` for Python
+- `go` for Golang
+
+If you would like to add an example in a different language, see the FAQ section for [adding support for your favorite language](https://www.pulumi.com/docs/troubleshooting/faq/#how-can-i-add-support-for-my-favorite-language).
+
+Each folder contains files related to your [Pulumi project](https://www.pulumi.com/docs/intro/concepts/project/), a README, an optional directory for your sample application (`www` for example), and an optional `images` directory if you're including images in your example.
+
+```
+|____README.md
+|____www
+|____images
+| |____deploy.gif
+| |____deploy.cast
+|____Pulumi.yaml
+|____package.json
+|____tsconfig.json
+|____index.ts
```

Comment:
Why do you have underscores in front of the file names?

Reply:
This is the output of `tree`. I've always liked using it for showing a directory structure.